Tools/scripts/build_options.py

```diff
@@ -236,6 +236,7 @@ def __init__(self,
     Feature('Airspeed Drivers', 'UAVCAN_ASPD', 'AP_AIRSPEED_UAVCAN_ENABLED', 'ENABLE UAVCAN AIRSPEED', 0, 'AIRSPEED'),   # NOQA: E501
 
     Feature('Actuators', 'Volz', 'AP_VOLZ_ENABLED', 'Enable Volz Protocol', 0, None),
+    Feature('Actuators', 'Volz_DRONECAN', 'AP_DRONECAN_VOLZ_FEEDBACK_ENABLED', 'Enable Volz DroneCAN Feedback', 0, None),
```

Comment:
needs matching extract features

Reply:
I'll follow-up with you and @peterbarker about it.

Reply:
updated!